src/shared/components/lazyMobXTable/styles.scss

```diff
@@ -27,3 +27,14 @@ th.sort-asc:after {
     margin-bottom: 5px;
   }
 }
+
+.table td.text {
```

Comment:
are we sure we want to make this change for *every* LazyMobxTable? sure you dont want to just put this into the coexpression tab tables css?

Reply:
Agreed, this shouldn't be here, and it is not necessary, so I've removed it.